--- bin/wrapper.sh~	2009-01-24 12:04:23.000000000 +0100
+++ bin/wrapper.sh	2010-12-13 13:24:59.000000000 +0100
@@ -54,8 +54,8 @@
 	;;
     m2vcut_outputhelp)
     	f=`echo $LANG |sed 's/[^A-Za-z_].*//'`
-	pd=`cd \`dirname "$0"\`/..; pwd`
-	helpprefix="$pd/doc/m2vcut_help"
+	pd=`cd \`dirname "$0"\`/../../..; pwd`
+	helpprefix="$pd/share/m2vmp2cut/m2vcut_help"
 	test -f "$helpprefix-$f" && cat "$helpprefix-$f" || cat "$helpprefix-default"
 	read line
 	;;
--- bin/m2vmp2cut.pl~	2009-02-07 21:54:24.000000000 +0100
+++ bin/m2vmp2cut.pl	2010-12-13 13:24:55.000000000 +0100
@@ -43,7 +43,7 @@
 	open STDOUT, ">&STDERR" || warn "1>&2 redirection failed: $! !!!\n";
     }
 
-    system 'cat', "$M2VMP2CUT_CMD_PATH/../doc/$_[1]";
+    system 'cat', "$M2VMP2CUT_CMD_PATH/../../../share/m2vmp2cut/$_[1]";
 
     exit $_[0];
 }
--- Makefile~	2010-12-13 12:05:16.000000000 +0100
+++ Makefile	2010-12-13 12:14:26.000000000 +0100
@@ -35,8 +35,8 @@
 	m2vmp2cut.pl m2vmp2cut.sh m2v_catfiltered.py m2v_somehdrinfo.py \
 	wrapper.sh lvev6frames.pl getyuv.pl getmp2.sh m2vmp2cut.pm
 
-#TRG_DOCS = Examples m2vcut_help-default m2vcut_help-fi_FI Options Usage
-TRG_DOCS = m2vcut_help-default m2vcut_help-fi_FI Options Usage
+TRG_DOCS = Examples m2vcut_help-default m2vcut_help-fi_FI Options Usage
+#TRG_DOCS = m2vcut_help-default m2vcut_help-fi_FI Options Usage
 
 chkfiles: all
 	@for i in $(TRG_BINS:%=bin/%) $(TRG_DOCS:%=doc/%); \
@@ -48,10 +48,10 @@
 	@chkdir() { [ -d "$$1" ] || mkdir -m 755 -p "$$1"; }; \
 	chkdir $(PREFIX)/libexec/m2vmp2cut-$(VER); chkdir $(PREFIX)/bin;  \
 	chkdir $(PREFIX)/libexec/m2vmp2cut-$(VER)/bin; \
-	chkdir $(PREFIX)/lib/m2vmp2cut-$(VER)/doc; \
+	chkdir $(PREFIX)/share/m2vmp2cut; \
 	chkdir $(PREFIX)/libexec/m2vmp2cut-$(VER)/contrib
 	cp $(TRG_BINS:%=bin/%) $(PREFIX)/libexec/m2vmp2cut-$(VER)/bin
-	cp $(TRG_DOCS:%=doc/%) $(PREFIX)/lib/m2vmp2cut-$(VER)/doc
+	cp $(TRG_DOCS:%=doc/%) $(PREFIX)/share/m2vmp2cut
 	cp contrib/* $(PREFIX)/libexec/m2vmp2cut-$(VER)/contrib
 	cp m2vmp2cut $(PREFIX)/bin/m2vmp2cut
 	@test x'$(PJXJAR)' = x || \
